### ISSUE TITLE
Add votedTextstyle and OptionTextStyle

### ISSUE THREAD

**Issue by mihirdaka** [thenifemi/flutter_polls#33]
It would be nice to have textstyle for voted and non voted option

**Comment by looped-monk** [comment]
Yes. important feature. The colors of the text for voted should be different than the one for non-voted. The hasVoted boolean can be used to switch between voted and not voted.

**Comment by looped-monk** [comment]
> It would be nice to have textstyle for voted and non voted option

Hi. Any update on this feature mihir ?

**Comment by mihirdaka** [comment]
Hey,
I was able to achieve this but I have forked the repo and made some requirement-specific changes.
@thenifemi  I would appreciate it if you can review my changes and merge the PR

**Comment by thenifemi** [comment]
@mihirdaka Hey! That's great! Could you please open the PR and tag me in it?

**Comment by looped-monk** [comment]
@thenifemi @mihirdaka Is this PR complete on this package ?.

**Comment by mihirdaka** [comment]
Sorry just caught up with some deadlines. I will raise it as soon as I can

Mihir Daka

**Comment by narayanan-ka** [comment]
Ok.

**Comment by looped-monk** [comment]
@mihirdaka Hey. just checked in to see the updated PR.
